### PR TITLE
creates /api/version/deliveryservices/request api endpoint

### PR DESCRIPTION
**Pull request by mitchell852**
used to send the details of a new delivery service to a specified email address

### CODE REVIEW

File: docs/source/development/traffic_ops_api/v11/deliveryservice.rst [hunk 1]
```diff
@@ -1033,3 +1033,143 @@ SSL Keys
...
+          "maxLibrarySizeEstimate": "less-than-200-GB",
+          "originUrl": "http://myorigin.com",
+          "originDynamicRemap": false,
+          "originTestFile": "http://myorigin.com/crossdomain.xml",
+          "originAclWhitelist": true,
+          "originHeaders": "",
+          "otherOriginSecurity": "",
+          "queryStringHandling": "ignore-in-cache-key-and-pass-up",
+          "rangeRequestHandling": "range-requests-not-used",
+          "signedURLs": true,
```
Comment (dewrich) [inline docs/source/development/traffic_ops_api/v11/deliveryservice.rst]: seems inconsistent originUrl versus signedURLs, 'Url' or 'URL'? We are beginning to lean toward 'URL'

**Comment by knutsel** [comment]
I like Url better... it's easier when genning code and using tools to convert to camelCase to do everything with one leading caps.

File: docs/source/development/traffic_ops_api/v11/deliveryservice.rst (hunk 1 above)
Comment (dewrich) [inline docs/source/development/traffic_ops_api/v11/deliveryservice.rst]: 'signedURLs' looks like a boolean, which should become 'hasSignedURLs', to make it feel more grammatical

File: docs/source/development/traffic_ops_api/v11/deliveryservice.rst [hunk 2]
```diff
@@ -1033,3 +1033,143 @@ SSL Keys
...
+          "deliveryProtocol": "http",
+          "routingType": "dns",
+          "serviceDesc": "service description goes here",
+          "peakBpsEstimate": "less-than-5-Gbps",
+          "peakTpsEstimate": "less-than-1000-TPS",
+          "maxLibrarySizeEstimate": "less-than-200-GB",
+          "originUrl": "http://myorigin.com",
+          "originDynamicRemap": false,
+          "originTestFile": "http://myorigin.com/crossdomain.xml",
+          "originAclWhitelist": true,
```
Comment (dewrich) [inline docs/source/development/traffic_ops_api/v11/deliveryservice.rst]: Same boolean convention change here

File: docs/source/development/traffic_ops_api/v11/deliveryservice.rst [hunk 3]
```diff
@@ -1033,3 +1033,143 @@ SSL Keys
...
+          "customer": "XYZ Corporation",
+          "contentType": "video-on-demand",
+          "deliveryProtocol": "http",
+          "routingType": "dns",
+          "serviceDesc": "service description goes here",
+          "peakBpsEstimate": "less-than-5-Gbps",
+          "peakTpsEstimate": "less-than-1000-TPS",
+          "maxLibrarySizeEstimate": "less-than-200-GB",
+          "originUrl": "http://myorigin.com",
+          "originDynamicRemap": false,
```
Comment (dewrich) [inline docs/source/development/traffic_ops_api/v11/deliveryservice.rst]: Same boolean convention change here

File: docs/source/development/traffic_ops_api/v11/deliveryservice.rst [hunk 4]
```diff
@@ -1033,3 +1033,143 @@ SSL Keys
...
+       "details": {
+          "customer": "XYZ Corporation",
+          "contentType": "video-on-demand",
+          "deliveryProtocol": "http",
+          "routingType": "dns",
+          "serviceDesc": "service description goes here",
+          "peakBpsEstimate": "less-than-5-Gbps",
+          "peakTpsEstimate": "less-than-1000-TPS",
+          "maxLibrarySizeEstimate": "less-than-200-GB",
+          "originUrl": "http://myorigin.com",
```
Comment (dewrich) [inline docs/source/development/traffic_ops_api/v11/deliveryservice.rst]: originURL

**Comment by dewrich** [comment]
Here's where we will have issues in the future: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

**Pull request merged**